tests/licensedcode/data/licenses/nysl-0.9982_en.yml

```diff
@@ -0,0 +1,2 @@
+licenses:
+    - nysl-0.9982_en
```

Comment:
This line could be the cause of the `AssertionError`. Try replacing this with `nysl-0.9982`